.clang-format

```diff
@@ -34,4 +35,4 @@ BinPackArguments: true
 BinPackParameters: true
 ConstructorInitializerAllOnOneLineOrOnePerLine: false
 
-IndentCaseLabels: true
```

Comment:
Nit: newline

Reply:
There should be a newline. :) I think Git only shows the lack of a newline (see red/removed area vs. green/added area).